tests/Mailer/Ownership/SendMailsForOwnershipEventHandlerTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership;
+
+use CultuurNet\UDB3\Broadway\Domain\DomainMessageSpecificationInterface;
+use CultuurNet\UDB3\CommandHandling\ContextDecoratedCommandBus;
+use CultuurNet\UDB3\EventSourcing\DomainMessageBuilder;
+use CultuurNet\UDB3\Mailer\Command\SendOwnershipRequestedMail;
+use CultuurNet\UDB3\Ownership\Events\OwnershipRequested;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class SendMailsForOwnershipEventHandlerTest extends TestCase
```

Comment:
Maybe we also need a test to make sure no mails are send for another random event then `OwnershipRequested` ?